common.tf

```diff
@@ -168,3 +170,59 @@ module "files_table" {
     }
   ]
 }
+
+data "aws_ssm_parameter" "slack_token" {
+  name            = "/mgmt/slack/token"
+  with_decryption = true
+}
+
+data "aws_ssm_parameter" "dr2_notifications_slack_channel" {
+  name = "/mgmt/slack/notifications/channel"
```

Comment:
Should there be some sort of `${local.environment}` here instead of `mgmt` or is this fine?

Reply:
Same as above